spec/models/card_spec.rb

```diff
@@ -0,0 +1,105 @@
+require 'rails_helper'
+
+describe Card do
+  it 'create card with empty original text' do
+    card = Card.create(original_text: '', translated_text: 'house')
+    expect(card.errors[:original_text]).to include('Необходимо заполнить поле.')
+  end
+
+  it 'create card with empty translated text' do
+    card = Card.create(original_text: 'дом', translated_text: '')
+    expect(card.errors[:translated_text]).
+        to include('Необходимо заполнить поле.')
+  end
+
+  it 'create card with empty texts' do
+    card = Card.create(original_text: '', translated_text: '')
+    expect(card.errors[:original_text]).
+        to include('Вводимые значения должны отличаться.')
+  end
+
+  it 'equal_texts Eng' do
+    card = Card.create(original_text: 'house', translated_text: 'house')
+    expect(card.errors[:original_text]).
+        to include('Вводимые значения должны отличаться.')
+  end
+
+  it 'equal_texts Rus' do
+    card = Card.create(original_text: 'дом', translated_text: 'дом')
+    expect(card.errors[:original_text]).
+        to include('Вводимые значения должны отличаться.')
+  end
+
+  it 'full_downcase Eng' do
+    card = Card.create(original_text: 'hOuse', translated_text: 'houSe')
+    expect(card.errors[:original_text]).
+        to include('Вводимые значения должны отличаться.')
```

Comment:
Use 2 (not 4) spaces for indenting an expression spanning multiple lines.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.